packages/compass-e2e-tests/index.js

```diff
@@ -35,13 +35,18 @@ async function main() {
     mocha.addFile(path.join(__dirname, testPath));
   });
 
-  const failures = await promisify(mocha.run.bind(mocha))();
-
-  process.exitCode = failures ? 1 : 0;
+  try {
+    const failures = await promisify(mocha.run.bind(mocha))();
+    process.exitCode = failures ? 1 : 0;
+  }
+  catch (err) {
+    console.error('mocha threw', err);// we never see failures. mocha throws 1
```

Comment:
Need to figure out what the right thing is to do here. For some reason many test failures don't exit the tests and it just hangs there with compass still open. Not sure if it is this case or the unhandledRejection one below or both.